src/packages.lisp

```diff
@@ -39,7 +39,8 @@
    #:display-svg render-svg
    #:display-json render-json
    #:display-javascript render-javascript
-   #:kernel-start))
+   #:kernel-start
+   #:kernel-start-exev))
```

Comment:
kernel-start-exev appears to be a typo -- is it supposed to be kernel-start-exec ??

Reply:
Thanks! Good catch.